pandas/core/arrays/base.py

```diff
@@ -617,42 +617,60 @@ def _ndarray_values(self):
         return np.array(self)
 
 
-class ExtensionDefaultOpsMixin(object):
+
+class ExtensionOpsMixin(object):
     """
-    A base class for default ops, that returns NotImplemented
+    A base class for linking the operators to their dunder names
     """
     @classmethod
-    def _create_method(cls, op, coerce_to_dtype=True):
-        """
-        A class method that returns a method will correspond to an
-        operator for an ExtensionArray subclass, by always indicating
-        the operator is NotImplemented
-
-        Parameters
-        ----------
-        op: function
-            An operator that takes arguments op(a, b)
-        coerce_to_dtype:  bool
-            boolean indicating whether to attempt to convert
-            the result to the underlying ExtensionArray dtype
-            (default True)
-
-        Returns
-        -------
-        A method that can be bound to a method of a class
-        """
-
-        def returnNotImplemented(self, other):
-            return NotImplemented
+    def addArithmeticOps(cls):
+        cls.__add__ = cls._create_method(operator.add)
```

Comment:
name these like

_add_numeric_ops and so on

Reply:
Fixed